Makefile

```diff
@@ -874,7 +874,7 @@ CNI_TYPE ?= cilium
 
 # COVER_FILTER omits folders with all files tagged with one of 'unit', '!ignore_uncovered', or '!ignore_autogenerated'
 test-all: ## run all unit tests.
-	@$(eval COVER_FILTER=`go list --tags ignore_uncovered,ignore_autogenerated $(COVER_PKG)/... | tr '\n' ','`)
+	@$(eval COVER_FILTER=`go list $(COVER_PKG)/... | tr '\n' ','`)
```

Comment:
everything else looks fine, but what is the effect of this change?

Reply:
I think this will affect the code coverage % somewhat but this was causing one of the flaky tests to be fail repeatedly 
https://teams.microsoft.com/l/message/19:2b104ee19fdb450b807903eb7c79588a@thread.skype/1724175877025?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=1eb27b57-8ace-4f0e-a9a9-e6db4e9829b7&parentMessageId=1724175877025&teamName=ContainerNetworking&channelName=CNI&createdTime=1724175877025

Reply:
@rbtr  I checked the coverage % diff and its negligible. 
Without makefile change: https://msazure.visualstudio.com/One/_build/results?buildId=101038052&view=logs&j=792982d5-3bb1-5d82-222e-228148b73448&t=fe31bb4d-d8c6-5505-3ab3-3c9a2560c42b

With Makefile change:  https://msazure.visualstudio.com/One/_build/results?buildId=101344151&view=logs&j=792982d5-3bb1-5d82-222e-228148b73448&t=fe31bb4d-d8c6-5505-3ab3-3c9a2560c42b

Reply:
can we update the comment too to reflect what we're including in testing/coverage

Reply:
updated